### PR TITLE
net.mediaarea.MediaConch

### PR DESCRIPTION
**Pull request by g-maxime**
Tested with flatpak-builder v0.10.3 and flatpak v0.10.2.1 on Gentoo x86_64

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build net.mediaarea.MediaConch

**Comment by nedrichards** [comment]
It'd be great to send the appdata upstream if possible.

File: net.mediaarea.MediaConch.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2016-2018 MediaArea.net <info@mediaarea.net> -->
+<component type="desktop">
+  <id type="desktop">net.mediaarea.MediaConch.desktop</id>
+  <metadata_license>CC-BY-3.0</metadata_license>
```
Comment (nedrichards) [inline net.mediaarea.MediaConch.appdata.xml]: CC0 is the preferred licence here.

**Comment by nedrichards** [comment]
Looks like it doesn't like the appdata location.

**Comment by g-maxime** [comment]
@nedrichards Thank, fixed and all PR updated with CC0 licence.

File: net.mediaarea.MediaConch.json [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+          "sha256": "795ceec2251071916fe48d58d9cd2274f6ce9ede608d06cbb6b5418640b45b25"
+        },
+        {
+            "type": "file",
+            "path": "net.mediaarea.MediaConch.appdata.xml"
+        }
+      ],
+      "post-install": [
+        "install -D -m 0644 ../GNU/GUI/mediaconch-gui.desktop /app/share/applications/mediaconch-gui.desktop",
+        "install -D -m 0644 ../../Source/Resource/Image/MediaConch.png /app/share/icons/hicolor/256x256/apps/mediaconch.png",
```
[the remark this answers is not in the document]
Comment (g-maxime) [reply net.mediaarea.MediaConch.json]: The autogenerated icons in share/app-info/icons are not enough?

[the remark this answers is not in the document]
Comment (g-maxime) [reply net.mediaarea.MediaConch.json]: @nedrichards PR updated.

**Comment by g-maxime** [comment]
@nedrichards PR updated.

**Comment by nedrichards** [comment]
bot, build net.mediaarea.MediaConch

File: net.mediaarea.MediaConch.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (nedrichards) [reply net.mediaarea.MediaConch.json]: If the 64x64 and 128x128 are correctly exported, that's fine.

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/net.mediaarea.MediaConch

**Pull request closed without merging**